src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +208,230 @@
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="0" Grid.Column="2" BorderThickness="1,0,0,0" BorderBrush="Black">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_LastSettingsRestoreDate" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="0" Grid.Column="3">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding LastSettingsRestoreDate, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+
+                                <Border Grid.Row="2" Grid.Column="0">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_LastSettingsBackupSource" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="1">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding LastSettingsBackupSource, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="2" BorderThickness="1, 0, 0, 0" BorderBrush="Black">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_CurrentSettingMatchStatus" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="3">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding CurrentSettingMatchText, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+                            </Grid>
+
+
+
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
+                </controls:SettingsGroup>
+
+
+                <InfoBar x:Uid="General_SettingsBackupMessageResults"
+                                  IsClosable="False"
+                                  Severity="{Binding BackupRestoreMessageSeverity, Converter={StaticResource StringToInfoBarSeverityConverter}}"                         
+                                  IsTabStop="False"
```

Comment:
@niels9001, don't we need true here for narrator?

Reply:
Hmm, maybe. In github, it show 4 lines of code. I have to assume, because you didn't say, you are referring to the last line of the four, `IsTabStop="False"`?

I will change to 

`IsTabStop="{Binding SettingsBackupRestoreMessageVisible, Mode=OneWay}"`

I will note that it seems like that and even `IsTabStop="True"` does not seem to allow the tab to navigate to the InfoBar elements, but I have made the suggestion nonetheless.